wandb/integration/metaflow/metaflow.py

```diff
@@ -223,6 +235,21 @@ def wandb_log(
     others=False,
     settings=None,
 ):
+    """
+    Automatically log parameters and artifacts to W&B by type dispatch.
+
+    This decorator can be applied to a flow, step, or both.
+    - Decorating a step will enable or disable logging for certain types within that step
+    - Decorating the flow is equivalent to decorating all steps with a default
+    - Decorating a step after decorating the flow will overwrite the flow decoration
+
+    Arguments:
+        datasets (bool): If `True`, log datasets.  Datasets can be a `pd.DataFrame` or `pathlib.Path`.  The default value is `False`, so datasets are not logged.
```

Comment:
@andrewtruong you'll [need to move the `:` to the other side of the type annotation](https://www.notion.so/wandbai/Docstring-Style-Guide-76d247fc6e0d46a3891bd4c37b3501c1#5c7c84ea12144c9db83ac7ec32146bca) (so `datasets: (bool). If ...`). This is a kind of annoying requirement of our .

You could alternatively add type annotations to the function definition, and then you wouldn't need them here. That's the direction we'd like to move the library. If you go this path, ask someone on client for details about the mypy setup.

Reply:
Done